content/tutorials/manage-the-rippled-server/configuration/configure-advisory-deletion.md

```diff
@@ -0,0 +1,96 @@
+# Configure Advisory Deletion
+
+In its default configuration, the `rippled` server automatically deletes outdated history of XRP Ledger state and transactions as new ledger versions become available. If your server uses most of its hardware resources during peak hours, you can configure the server to delete ledgers only when prompted by a command scheduled to run during off-peak hours to reduce the chances of online deletion
```

Comment:
By "default" rippled does not do online_delete. It needs to be configured. However, the example config file shipped right now with the source code happens to configure online_delete.